src/BullOak.Repositories.EventStore/Events/SoftDeleteEvent.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace BullOak.Repositories.EventStore.Events
+{
+    public class SoftDeleteEvent
```

Comment:
I would suggest using past tense for event, maybe `EntitySoftDeleted`?

Reply:
Ah yes, just because it's an implementation detail doesn't exempt this from the rule that events should be in the past tense. Well spotted!